routes/modules.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use App\Http\Livewire\ModuleContent;
+
+Route::multilingual('/modules/{module}', [ModuleContent::class, '__invoke'])
```

Comment:
should probably be under courses. that is `/courses/{course}/{module}` or `/courses/{course}/modules/{module}`

Reply:
decided to use the first route.

Reply:
Hey @jobara, I couldn't figure out how I can do route model binding for {course} from modules route. Could you help me on this please?

Reply:
@chosww could you explain a bit more of what the issue you are running into is? What part isn't working. I'm not sure if it's an issue with the livewire component for example or maybe the way you are calling the route?

For example when you're [accessing the route](https://github.com/accessibility-exchange/platform/blob/1325a300f43de162e2e645f8b5bbb6d06435da81/resources/views/courses/show.blade.php#L46) you'll need to pass in the course instance. 

Something like:

```php
<a href="{{ localized_route('modules.module-content', ['course' => $module->course, 'module' => $module]) }}">
    {{ $module->title }}
</a>
```

You may also want to look at [scoped bindings](https://laravel.com/docs/9.x/routing#implicit-model-binding-scoping).

Reply:
If you're still running into issues, it may be something that needs to be updated in the livewire `mount` to take in the `course` or you may need to look at this livewire issue, https://github.com/livewire/livewire/issues/2558, where they talk about explicitly setting the model binding.

Reply:
Thanks Justin, I was able to resolve the issue I was having with your first suggestion.